director/redirect.go

```diff
@@ -223,8 +223,8 @@ func RedirectToCache(ginCtx *gin.Context) {
 			ginCtx.Writer.Header()["X-Pelican-Token-Generation"] = []string{tokenGen}
 		}
 	}
-	ginCtx.Writer.Header()["X-Pelican-Namespace"] = []string{fmt.Sprintf("namespace=%s, require-token=%v",
-		namespaceAd.Path, namespaceAd.RequireToken)}
+	ginCtx.Writer.Header()["X-Pelican-Namespace"] = []string{fmt.Sprintf("namespace=%s, require-token=%v, dirlisthost=%s",
```

Comment:
I know that for topology, we've committed to calling this `dirlisthost`.  However, that seems like a really bad name for Pelican (as we don't make any assumptions about a directory!).

How about `collections-url`?